test/Math/Rectangle.test.cpp

```diff
@@ -61,6 +61,34 @@ TEST(Rectangle, translate) {
 	EXPECT_EQ((NAS2D::Rectangle{4, 6, 5, 6}), (NAS2D::Rectangle{3, 4, 5, 6}).translate({1, 2}));
 }
 
+TEST(Rectangle, operatorAddSubtract) {
+	//translate(...) and operator overloads are equivalent.
+	EXPECT_EQ((NAS2D::Rectangle{0, 0, 1, 1}).translate({1, 1}), (NAS2D::Rectangle{0, 0, 1, 1} + Vector<int>{1, 1}));
+	EXPECT_EQ((NAS2D::Rectangle{1, 1, 1, 1}).translate({1, 1}), (NAS2D::Rectangle{1, 1, 1, 1} + Vector<int>{1, 1}));
+	EXPECT_EQ((NAS2D::Rectangle{3, 4, 5, 6}).translate({1, 2}), (NAS2D::Rectangle{3, 4, 5, 6} + Vector<int>{1, 2}));
```

Comment:
Unit tests typically make use of CTAD (Class Template Argument Deduction), so the `<int>` is unnecessary in most cases. It should be removed for consistency.